<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionEquipements</class>
 <widget class="QWidget" name="GestionEquipements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimer">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/logo/delete.png</normaloff>:/logo/delete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="nom_t">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nueva Std</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gestion des equipements</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icons8_add_new_96px.png</normaloff>:/images/icons8_add_new_96px.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_t">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Numero">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Subheading</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Formulaire d'ajout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>etat</string>
   </property>
  </widget>
  <widget class="QLabel" name="etat_t">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="numero_t">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Salle</string>
   </property>
  </widget>
  <widget class="QComboBox" name="salles">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherche_value">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="placeholderText">
    <string>rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icons8_search_more_24px_1.png</normaloff>:/images/icons8_search_more_24px_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="modifier">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icons8_update_24px.png</normaloff>:/images/icons8_update_24px.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icons8_synchronize_24px_1.png</normaloff>:/images/icons8_synchronize_24px_1.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Subheading</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>et de modification</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stats">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icons8_combo_chart_100px.png</normaloff>:/images/icons8_combo_chart_100px.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="etat">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PDF">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>Generer PDF</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/logo/delete.png</normaloff>:/logo/delete.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
